extensions/interactions/NumericInput/protractor.js

```diff
@@ -32,6 +32,8 @@ var expectInteractionDetailsToMatch = function(elem) {
 var submitAnswer = function(elem, answer) {
   elem.element(by.tagName('oppia-interactive-numeric-input')).
     element(by.tagName('input')).sendKeys(answer + '\n');
```

Comment:
Do you still need \n?

It is a little irritating to not be able to submit an answer by pressing enter. I think we will want to reinstate that functionality somehow, actually. How problematic would that be? (At this point I can't actually remember the original reason we wanted to get rid of the form...)